Profile Creation

Treannataylor123 opened the pull request:
I have added the code for both the .xml layout and .java for the profile creation. I am not sure if the spring boot was set up so I am unable to add the route for the profile. Please review.

Commit:
Profile Creation

finalfantasyfan87 reviewed (changes requested):
Everything looks great, but I think we shouldn't include the .idea, .gradle directory or any binaries. We should only check in the .java and xml layout files. Let me know anyway I can help with the clean up

Commit:
disregard the gradle build

finalfantasyfan87 reviewed (dismissed):
looks good to me
lets remove commented out code, remove  files in .idea directory and im checking whether or not we should check in gradle files
other than that, nice job

Commit:
adding some permissions and sample api request

adding some permissions and sample api dummy request to treanna's existing work
